features/works/work_edit.feature

```diff
@@ -131,4 +131,28 @@ Feature: Edit Works
       And I post the work "Load of Typos"
     When I edit the work "Load of Typos"
       And I press "Preview"
-    Then I should not see "draft"
\ No newline at end of file
+    Then I should not see "draft"
+
+  Scenario: You can add a co-author to an already-posted work
+    Given I am logged in as "leadauthor"
+      And I post the work "Dialogue"
+    When I add the coauthor "coauthor" to the work "Dialogue"
```

Comment:
It looks like one of the test failures is happening because there's a hyphen missing in "add the co-author"